### PR TITLE
Updates to badge admin pages

### PR DESCRIPTION
**Pull request by alyssadelaine**
Closes https://github.com/developmentseed/scoreboard/issues/133

* Adds a carousel with available badge images for admin to choose from at time of badge creation
* The same carousel (scrolled to the previous image) is available to choose a new image when editing a badge
* Allows an admin to enter the name of a hashtag and create a badge for users who have edited using that hashtag
* Changes the form for hashtag badges since "Condition" and "Number" aren't meaningful in that case

### CODE REVIEW

File: lib/utils/loadImages.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+function createImageList () {
```
Comment (kamicut) [inline lib/utils/loadImages.js]: This is pretty opaque, what are you doing here? Could you add a comment?

Comment (alyssadelaine) [reply lib/utils/loadImages.js]: done

**Comment by kamicut** [comment]
@scisco @sethvincent could you try this in your browsers? @alyssadelaine and I can get it working on Chrome and some version of Firefox, but I couldn't get it running in my current Firefox.

**Comment by sethvincent** [comment]
Worked for me in firefox, chrome, and safari.

We should move the "add another condition" button to be above the image selection. It's a little confusing being separated from the existing conditions.

**Pull request merged**